patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/manage-tModLoaderServer.sh

```diff
@@ -49,21 +42,25 @@ function update_script {
 }
 
 function get_latest_release {
-	local latest_release
-	latest_release=$(curl --silent "https://api.github.com/repos/tModLoader/tModLoader/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/') # Get latest release from github's api
+	local 
+release_url="https://api.github.com/repos/tModLoader/tModLoader/releases/latest"
```

Comment:
```suggestion
	local release_url="https://api.github.com/repos/tModLoader/tModLoader/releases/latest"
```
Please fix the formatting - this doesn't need a new line.

Everything else looks good, and seems to run ok, 
Great work making a clean fix.